### PR TITLE
chore: remove delphi, carthage and babylon integration tests

### PR DESCRIPTION
**Pull request by jevonearth**
Replaces #737

### CODE REVIEW

File: integration-tests/batch-api.spec.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ CONFIGS().forEach(({ lib, rpc, setup, knownBaker, createAddress }) => {
     })
 
     it('Simple transfers with origination using with', async (done) => {
-      const op = await Tezos.batch([
+      const op = await Tezos.contract.batch([
```
Comment (roxaneletourneau) [inline integration-tests/batch-api.spec.ts:32]: Should we keep this test as it until we removed the deprecated batch function?

contract.batch() is covered in the file contract-batch.spec.ts

File: integration-tests/contract-originate-contract-with-sapling.spec.ts [hunk 2]
```diff
@@ -4,17 +4,16 @@ import { Protocols } from "@taquito/taquito";
...
-  
   describe(`Test origination of contracts with sapling using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup();
       done()
     })
-
-     edonet('Originates a contract having an empty sapling state in its storage', 2, async (done: () => void) => {
+    //  deepcode ignore only-arrow-functions: not sure how to do it otherwise
```
Comment (roxaneletourneau) [inline integration-tests/contract-originate-contract-with-sapling.spec.ts:15]: Do you think this line can be removed if the following one is written this way: 
edonet('Originates a contract having an empty sapling state in its storage', 2, async (done: () => void) => {

File: integration-tests/edo-deploy-contract-having-ticket.spec.ts [hunk 3]
```diff
@@ -4,17 +4,16 @@ import { Protocols } from "@taquito/taquito";
...
 
   describe(`Test origination of a token contract using: ${rpc}`, () => {
 
     beforeEach(async (done) => {
       await setup();
       done()
     })
-
-    edonet('Originates a contract having ticket with init and the wallet api', 2, async (done: () => void) => {
+    //  deepcode ignore only-arrow-functions: not sure how to do it otherwise
```
Comment (roxaneletourneau) [inline integration-tests/edo-deploy-contract-having-ticket.spec.ts:15]: Same as above

File: integration-tests/contract-originate-contract-with-sapling.spec.ts (hunk 2 above)
Comment (michaelkernaghan) [reply integration-tests/contract-originate-contract-with-sapling.spec.ts:15]: i will change it

Comment (michaelkernaghan) [reply integration-tests/contract-originate-contract-with-sapling.spec.ts:15]: made the change

File: integration-tests/edo-deploy-contract-having-ticket.spec.ts (hunk 3 above)
Comment (michaelkernaghan) [reply integration-tests/edo-deploy-contract-having-ticket.spec.ts:15]: i will try it

**Pull request merged**